docs/store-operations/pricing/msf-international-enhancements.mdx

```diff
@@ -0,0 +1,93 @@
+# Price Lists - International Enhancements for Multi-Storefront
+
+<Callout type="info">
+  Tax-inclusive and exclusive price lists are in beta. Features may change based on user feedback or internal testing.
+</Callout>
+
+When you sell internationally, you may need your storefront to display prices that either include or exclude tax based on regional expectations. You may need some storefront channels to display prices that include tax, while others display prices that exclude tax.
+
+To localize your storefronts, you can do the following:
+- Specify whether a price list has prices that include or exclude tax when entered.
+- Assign the price list to storefront channels (or customer groups).
+
+
+## Limitations
+
+- Stores using [Catalog V2](/docs/store-operations/catalog/migration) have fixed pricing and will use global store settings. 
+- You cannot enter price changes due to product modifier rules, gift wrapping features, and fixed shipping prices into a price list. These inherit global settings and always apply on top of the price list.
+- You cannot mix inclusive and exclusive pricing within a single price list.
+- You must [set the display price](https://support.bigcommerce.com/s/article/Product-Display-Settings?language=en_US#settings) to the lowest variant price when a shopper's preselected options don't match to a price. 
+
+If your store doesn't have multiple storefronts, you can still create tax-inclusive or exclusive price lists and assign them to customer groups.
+
```

Comment:
We should also add something about going into Settings > Tax > Store Tax Settings and turning on `Keep prices as entered` for `When calculating the taxes on sales made outside the pricing tax zone `

_especially_ for folks on default ex tax stores

Reply:
I see this below now :-)